client/opentracing/src/main/java/io/opentracing/impl/APMSpan.java

```diff
@@ -435,4 +435,8 @@ protected TraceContext getTraceContext() {
         }
         return state;
     }
+
+    public String toString() {
+        return super.toString() + "[operationName=" + getOperationName() + " state=" + state() + "]";
```

Comment:
`super` does not define `toString()` if it's added we end up with two operation names.